src/server/svr_chk_owner.c

```diff
@@ -575,6 +575,10 @@ svr_authorize_resvReq(struct batch_request *preq, resc_resv *presv)
 	if ((preq->rq_perm & (ATR_DFLAG_OPRD | ATR_DFLAG_OPWR |
 		ATR_DFLAG_MGRD | ATR_DFLAG_MGWR)) != 0)
 		return (0);
+	/* Only Manager has privilage to force modify reservation */
+	if (preq->rq_type == PBS_BATCH_ModifyResv && (preq->rq_extend != NULL) &&
+	    (strcmp(preq->rq_extend, FORCE) == 0) && ((preq->rq_perm & ATR_DFLAG_MGWR) == 0))
```

Comment:
Should this be strstr instead of strcmp? What if we add more things to ralter -W, and "Force" isn't the first one?

Reply:
You are right, but we don't know what will be the next option name (maybe the new name has 'force' in it too) and if multiple options can be given within one extend field then I think we will have to break it on a delimiter and then do strcmp.
I guess this is fine for now since this is the only option. Please let me know if you think otherwise.

Reply:
Your explanation works for me